src/gd_kotlin.cpp

```diff
@@ -154,6 +167,9 @@ void GDKotlin::init() {
             is_gc_activated = false;
             //TODO: Link to documentation
             WARN_PRINT("GC thread was disable. --jvm-disable-gc should only be used for debugging purpose")
+        } else if (cmd_arg == "--jvm-disable-closing-leaks-warning") {
```

Comment:
Could we add a little doc to the readme about all the cmdline arguments we can pass in?

Reply:
Because there are quite a few now

Reply:
If you're ok I'll do it in a seperate PR, in which I will create a conf object on cpp side so that we can move command args parsing from GdKotlin singleton. This would improve maintainability. Maybe write an issue about this ?

Reply:
Ok I'll write one and assign it to you